src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,77 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+import React, { useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Do not forget commit and push package.json with required dependecies
![2024-03-04_15h50_44](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/61904995/ac7dea61-9cef-40df-b46e-cb594a6411ab)